tests/unit/preprocessor/_regrid/test_regrid.py

```diff
@@ -108,5 +117,46 @@ def test_regrid__cell_specification(self):
         self.assertEqual(set(_CACHE.keys()), set(specs))
 
 
+class TestCloseness(tests.Test):
+    def test_regrid__closeness_cell_specification(self):
+        latitude = iris.coords.DimCoord(np.linspace(-85, 85, 18),
+                                        standard_name='latitude',
+                                        units='degrees')
+        longitude = iris.coords.DimCoord(np.linspace(5, 355, 36),
+                                         standard_name='longitude',
+                                         units='degrees')
+        latitude2 = iris.coords.DimCoord(np.linspace(-85, 85, 17),
```

Comment:
Maybe you could give this coordinate the same shape as `latitude`, but different bounds? Because now only the first check (if the shape of the coordinates is the same) gets executed in the test, the second check (if the bounds are the same) is never executed in a test run.

Reply:
Yes you are Right. I only tested for shape. So I'd add 2 more cubes for longitude and latitude shifted Bounds, just to be complete?! Push a new test as soon as possible.

Reply:
Yes, that should be fine. Or you could just make one of the coordinates of equal shape but with different bounds, then you also have this case covered.

Reply:
I thought I would have to test that for both axes independently.

Reply:
The minimum amount of testing I would recommend is that at least all code is executed once by a test, that way you're sure all code actually works. You're using the same code for both axes, because it's in a loop, so I think that testing for only one of the axes in one of the test cases should be OK.

Of course more thorough testing is better, but it's good to think about the balance between the amount of effort you put in creating the tests and subsequently maintaining those tests and how sure you want to be that the code does what it is supposed to do.